config/main.py

```diff
@@ -1999,6 +2002,20 @@ def override_config_db(config_db, config_input):
     click.echo("Overriding completed. No service is restarted.")
 
 
+def yang_hard_dependency_check(config_json):
```

Comment:
>yang_hard_dependency_check [](http://example.com/codeflow?start=4&length=26)

Just carious if there is a generic solution which using yang-model to parse the ConfigDB, and detect the missing dependency issues only.